libs/designsystem/src/lib/components/icon/icon.component.spec.ts

```diff
@@ -47,6 +47,22 @@ describe('IconComponent', () => {
       expect(component.name).toBe(undefined);
       expect(component.icon.name).toBe(component.defaultIcon.name);
     });
+
+    it('should use custom icons from ICON_SETTINGS', () => {
+      const fixture = createTestComponent(`<kirby-icon customName="name1"></kirby-icon>`);
+      fixture.detectChanges();
+      const component = fixture.debugElement.query(By.directive(IconComponent)).componentInstance;
+      expect(component.icon.name).toBe('name1');
+      expect(component.icon.svg).toBe('svg1');
+    });
+
+    it('should use custom icons added with KirbyIconRegistryService', () => {
+      const fixture = createTestComponent(`<kirby-icon customName="customName1"></kirby-icon>`);
```

Comment:
Consider icon naming (to reflect the icon comes from a custom ICON_SETTINGS)
```suggestion
      const fixture = createTestComponent(`<kirby-icon customName="customIconFromIconRegistry"></kirby-icon>`);
```

Reply:
👍